ci-operator/step-registry/storage/create/csi-gcp-filestore/storage-create-csi-gcp-filestore-commands.sh

```
@@ -19,6 +19,25 @@ export STORAGECLASS_LOCATION=${SHARED_DIR}/filestore-sc.yaml
 export MANIFEST_LOCATION=${SHARED_DIR}/${TEST_CSI_DRIVER_MANIFEST}
 export CONNECT_MODE="DIRECT_PEERING"
 
+# Version comparison functions using sort -V
+# Copied from other test steps
+function version_ge() {
+  # Returns 0 (true) if $1 >= $2
+  [[ "$1" == "$2" ]] && return 0
+  [[ "$(printf '%s\n' "$2" "$1" | sort -V | head -n1)" == "$2" ]]
+}
+
+cp ${CLUSTER_PROFILE_DIR}/pull-secret pull-secret
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)